ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -63,6 +63,7 @@ StatusSectionLayout {
     property var dismissedReceivedRequestContactsModel
 
     required property bool isCentralizedMetricsEnabled
+    required property var localSettings
```

Comment:
Similarly as for `AppearanceView` particular settings should be taken by individual properties, and change requests propagated via signals from appearance view upstream. It's more verbose but easier to read and track, no `var` involved as well.

Reply:
Alright, will do